### PR TITLE
RDISCROWD-5703: Allow edit submission for admins, subadmins

### PR DESCRIPTION
**Pull request by dchhabda**
Files to review
static/js/tasks_browse.js
templates/projects/tasks_browse.webpack.ejs

Rest of the files included in PR so that they're latest as per .ejs and are not required to be rebuild locally when respective .ejs not modified.

### CODE REVIEW

File: templates/projects/tasks_browse.webpack.ejs [hunk 1]
```diff
@@ -134,6 +134,14 @@
             </button>
         </li>
         {% endif %}
+        {% if allow_taskrun_edit and request.args['view'] != 'tasklist' and (current_user.id in project.owners_ids or current_user.subadmin or current_user.admin) %}
+        <li>
+          <!-- <button class="btn btn-sm" id='btnRefresh' type="button" data-toggle="popover" data-trigger="focus" data-placement="top" data-constraints='focus' data-content="{{_('Please refresh the results after setting the desired filters.')}}" title="{{_('Search')}}"></button> -->
```
Comment (n00rsy) [inline templates/projects/tasks_browse.webpack.ejs:139]: Can this comment be removed?

**Review by n00rsy — APPROVED** [review]
lgtm

File: templates/projects/tasks_browse.webpack.ejs (hunk 1 above)
Comment (dchhabda) [reply templates/projects/tasks_browse.webpack.ejs]: updated. Thanks!

File: templates/projects/tasks_browse.html [hunk 2]
```diff
@@ -130,6 +134,14 @@
             </button>
         </li>
         {% endif %}
+        {% if allow_taskrun_edit and request.args['view'] != 'tasklist' and (current_user.id in project.owners_ids or current_user.subadmin or current_user.admin) %}
+        <li>
+          <!-- <button class="btn btn-sm" id='btnRefresh' type="button" data-toggle="popover" data-trigger="focus" data-placement="top" data-constraints='focus' data-content="{{_('Please refresh the results after setting the desired filters.')}}" title="{{_('Search')}}"></button> -->
```
[the remark this answers is not in the document]
Comment (dchhabda) [reply templates/projects/tasks_browse.html]: updated. Thanks!

**Pull request merged**